### DISCUSSION TITLE
Make canceled task yellow in DAG graph

### DISCUSSION THREAD

**Discussion by hankquan** [Netflix/conductor#2701] [Ideas]
Now, a canceled task is red in DAG graph which is hard to distinguish from failed task, an idea is to fill canceled task with another color, such as yellow.

**Comment by apanicker-nflx** [comment]
@peterlau Could you help with this?

**Comment by erikbrgr** [comment]
@howaric If you are building Conductor from source, this can be achieved by changing the colors in `ui/src/components/diagram/diagram.scss`. For example, I changed the color for In Progress to yellow, to distinguish between Pending tasks and In Progress tasks.

**Comment by peterlau** [comment]
https://github.com/Netflix/conductor/commit/1765a287a2746069770ad83067cc86ed64f6dfb4
